OrbitCore/TracepointEventBuffer.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CORE_TRACEPOINT_EVENT_BUFFER_H_
+#define ORBIT_CORE_TRACEPOINT_EVENT_BUFFER_H_
+
+#include <limits>
+#include <set>
+
+#include "BlockChain.h"
```

Comment:
Does it use BlockChain, SamplingProfiler or Threading?

Reply:
Only SamplingProfiler. Thanks for pointing that out!